src/main.rs

```diff
@@ -288,18 +287,17 @@ fn start_vmm(cmd_arguments: ArgMatches) {
     let api_evt = EventFd::new(EFD_NONBLOCK).expect("Cannot create API EventFd");
 
     let http_sender = api_request_sender.clone();
-
-    let seccomp_level = if let Some(seccomp_value) = cmd_arguments.value_of("seccomp") {
+    let seccomp = if let Some(seccomp_value) = cmd_arguments.value_of("seccomp") {
         match seccomp_value {
-            "true" => SeccompLevel::Advanced,
-            "false" => SeccompLevel::None,
+            "true" => true,
+            "false" => false,
```

Comment:
No we don't want to switch to a boolean model, as we'd like to introduce a third option to log the faulty syscalls.